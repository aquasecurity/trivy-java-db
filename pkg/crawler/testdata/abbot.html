<!DOCTYPE html>
<html><head>
<meta http-equiv="content-type" content="text/html; charset=windows-1252">
	<title>Central Repository: abbot</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<style>
body {
	background: #fff;
}
	</style>
</head>

<body>
	<header>
		<h1>abbot</h1>
	</header>
	<hr>
	<main>
		<pre id="contents">
<a href="../">../</a>
<a href="abbot/" title="abbot/">abbot/</a>                                                           -         -
		</pre>
	</main>
	<hr>


</body></html>