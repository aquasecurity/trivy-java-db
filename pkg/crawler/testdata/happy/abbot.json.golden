[
  {
    "v": "0.12.3",
    "1": "51d28a27d919ce8690a40f4f335b9d591ceb16e9"
  },
  {
    "v": "0.13.0",
    "1": "596d91e67631b0deb05fb685d8d1b6735f3e4f60"
  },
  {
    "v": "1.4.0",
    "1": "a2363646a9dd05955633b450010b59a21af8a423",
    "l": [
      "EPL"
    ]
  },
  {
    "v": "1.4.0-lite",
    "1": "0547ab037068afa2026925bd94bfb9fcfcec9761"
  }
]